linux_os/guide/system/permissions/files/permissions_important_account_files/file_permissions_backup_etc_gshadow/rule.yml

```diff
@@ -0,0 +1,38 @@
+documentation_complete: true
+
+title: 'Verify Permissions on Backup gshadow File'
+
+description: |-
+    {{{ describe_file_permissions(file="/etc/gshadow-", perms="0000") }}}
+
+rationale: |-
+    The <tt>/etc/gshadow-</tt> file is a backup of <tt>/etc/gshadow</tt>, and as such,
+    it contains group password hashes. Protection of this file is critical for system security.
+
+severity: medium
+
+identifiers:
+    cce@rhel7: 83572-8
+    cce@rhel8: 83573-6
+
+references:
+    cis@rhel7: 6.1.9
+    cis@rhel8: 6.1.9
+
+ocil_clause: '{{{ ocil_clause_file_permissions(file="/etc/gshadow-", perms="----------") }}}'
+
+ocil: |-
+    {{{ ocil_file_permissions(file="/etc/gshadow-", perms="----------") }}}
+
+template:
+    name: file_permissions
+    vars:
+        filepath: /etc/gshadow-
+        filemode: '0000'
+        filemode@debian8: '0640'
```

Comment:
I believe that if you make customizations per product, this should be also reflected in description / ocil.